Sources/ViewControllers/Settings/SettingsTableViewController.swift

```diff
@@ -25,15 +32,16 @@ class SettingsTableViewController: UITableViewController, MFMailComposeViewContr
     }
 
     override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
-        if section == 0 {
+        switch SettingsSection(rawValue: section) {
+        case .item?:
```

Comment:
I'm guessing we can do better this approach without unwrapped each case.

Let's create guard check with something like guard `let SettingsSection = Settings(rawValue: section) else { return "" }` and then use this value with the switch.

What do you think?

Reply:
@roger-tan Yes, I realised unwrapping each case was a bad approach. Guard statement will also take care of the default case.

Reply:
Looks much better. Thankyou!
Can you review it @roger-tan ?